docs/howto/add_relationship_links.md

```diff
@@ -0,0 +1,139 @@
+[Back to Guides](../README.md)
+
+# How to add relationship links
+
+ActiveModelSerializers offers you many ways to add links in your JSON, depending on your needs.
+Usually the most common use case for links is supporting nested resources.
+
+The following examples are in Rails but the logic is the same for any framework
+used.
+
+The following examples assume that you don't want to load any data for relationships (include param is empty),
+specifically the following Rails controller was used for those examples:
+
+```ruby
+class Api::V1::UsersController < ApplicationController
+  def show
+    render jsonapi: User.find(params[:id]),
+      serializer: Api::V1::UserSerializer,
+      include: []
+  end
+```
+
+### Links as an attribute of a resource
+**This is applicable to JSONAPI, JSON and Attributes adapters**
+
+You can just define an attribute in the resource, named `links` and return the
+links you want there. An example:
```

Comment:
@vasilakisfil since I have a particular vision of how I want docs to read, do you care/mind if I branch off your banch, and do some minor changes?